docs/10_protocols.md

```diff
@@ -51,7 +51,7 @@ type Person {
 let x : Hashable = new Person() in print(x.hash());
 ```
 
-Anywhere you can annotate a symbol with a type (variables, attributes, function,method and type arguments, and return values), you can also use a protocol. For the purpose of type inference, protocols are treated as types.
+Anywhere you can annotate a symbol with a type (variables, attributes, function, method, type arguments and return values), you can also use a protocol. For the purpose of type inference, protocols are treated as types.
```

Comment:
I think the original version was correct except for the missing space.

```suggestion
Anywhere you can annotate a symbol with a type (variables, attributes, function, method and type arguments, and return values), you can also use a protocol. For the purpose of type inference, protocols are treated as types.
```

Reply:
You are right, I didn't realize "arguments" refers to both "method arguments" and "type arguments".